beacon_node/network/src/network_beacon_processor/mod.rs

```diff
@@ -478,8 +480,32 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
         })
     }
 
+    /// Create a new `Work` event for some custody columns. `process_rpc_custody_columns` reports
+    /// the result back to sync.
+    pub fn send_rpc_custody_columns(
+        self: &Arc<Self>,
+        block_root: Hash256,
+        custody_columns: Vec<CustodyDataColumn<T::EthSpec>>,
+        seen_timestamp: Duration,
+        process_type: BlockProcessType,
```

Comment:
Passing the process type here seems a bit redundant since it's always going to be `SingleDataColumn`, I guess this is remain from parent lookup?